### DISCUSSION TITLE
How can we add conditional css classes?

### DISCUSSION THREAD

**Discussion by fablexis** [vueform/vueform#166] [Questions]
How can I add a css class based on the value of a specific field using the `schema` attr?

currently, in the docs, the only conditional thing that we can use is the ability to show or hide elements, that's called [Conditional rendering](https://vueform.com/docs/conditional-rendering), but what about having a conditional css class?

let's say that I have an input text and a toggle, and when I enable the toggle I want to add a css class called 'my-required-class', how can I reference the value of the field that I want to set that?

@adamberecz

**Comment by adamberecz** [comment]
You are right, actually this hasn't been covered by Vueform. I've released an update in `1.9.1` that should allow passing class modifiers as a `function` which can make things dynamic, eg. add classes conditionally.

Here's the new section in the docs related to it:
https://vueform.com/docs/styles-and-layout#conditional-classes
